pkg/cmd/util.go

```diff
@@ -702,3 +703,20 @@ func updateFlagsToOptions(interactive, skipPreview, yes bool) (backend.UpdateOpt
 		SkipPreview: skipPreview,
 	}, nil
 }
+
+// loadJSONConfigFile loads file with json configuration data and unmarshals into a map[string]*json.RawMessage object.
+func loadJSONConfigFile(filePath string) (map[string]*json.RawMessage, error) {
```

Comment:
Rather than having this function here, I am wondering if we should call the same function we use to load local config specified to the `--policy-pack-config` flag of `pulumi up/preview/watch`.

https://github.com/pulumi/pulumi/blob/61928f04e0524893266e091b4c509f9900bcd4c8/pkg/resource/analyzer/config.go#L31-L37

That way we have consistent behavior of loading local policy config from a file.

This existing function does some validation of the `EnforcementLevel` values, and also, as a convenience, allows specifying enforcement level for a policy like:

```json
{
    "some-policy": "advisory"
}
```

Rather than having to specify it like:

```json
{
    "some-policy": {
        "enforcementLevel": "advisory"
    }
}
```

This would require adding some code to convert `map[string]plugin.AnalyzerPolicyConfig` to `map[string]*json.RawMessage`. Probably the easiest way to do that is to loop through the map and `json.Marshal` each `plugin.AnalyzerPolicyConfig` value to bytes. The bytes can then be cast to `json.RawMessage`. Similar to what's done here:

https://github.com/pulumi/pulumi/blob/c28c6022472572c71875ef2e7e3d5a35f7ea3c96/pkg/backend/httpstate/client/client.go#L633-L641

Reply:
Ok. I will leverage that function so we can use the validations it has and then just convert  the types.